tests/transforms/test_transform_pipeline.py

```diff
@@ -132,15 +138,10 @@ def normalize_side_effect(value, dummya, dummyb):
     for manifest in expected_manifests:
         mask_save = call(Path(manifest['roi-mask-source-ref']), ANY,
                          transparency=0)
-        outline_save = call(Path(manifest['source-ref']), ANY,
-                            transparency=0)
-        full_outline_save = call(
-                Path(manifest['full-outline-source-ref']), ANY, transparency=0)
         max_save = call(Path(manifest['max-source-ref']), ANY)
         avg_save = call(Path(manifest['avg-source-ref']), ANY)
 
-        calls = [mask_save, outline_save, full_outline_save,
-                 max_save, avg_save]
+        calls = [mask_save, max_save, avg_save]
```

Comment:
I see why you are not checking these calls any more and that you're actually writing to a `tmp_path` with `Image.save()`. In place of asserting these calls, I think you should at least check that the expected files exist and potentially also read them and assert that the size and datatypes are as expected.

Reply:
The images aren't ever created in the test pipeline which is why I didn't initially add an assertion. I added an assertion that the add_scale function is called and add_scale.save() is called for every outline that is created. This falls in line with how Nick previously built the test.